src/app/Processor/CreateProcessorPage/CreateProcessorPage.tsx

```diff
@@ -204,6 +204,7 @@ const CreateProcessorPage = (): JSX.Element => {
             malformedTransformationTemplate={malformedTemplate}
             schemaCatalog={schemas}
             getSchema={getSchema}
+            goBackToInstance={goToInstance}
```

Comment:
Moved this outside `ProcessorEdit` because in theory it should be a standalone component, not containing logic tied to specific page URLs.